blockchain/src/blockchain/verify.rs

```diff
@@ -45,9 +45,9 @@ impl Blockchain {
         }
 
         // Verify the interlink (or its absence)
-        if let BlockHeader::Macro(macro_header) = block.header() {
-            if block.is_election() {
-                if let Some(interlink) = &macro_header.interlink {
+        if let Block::Macro(macro_) = &block {
```

Comment:
Are the `macro_`s accidentally happen from refactoring?

Reply:
In the rest of the codebase, we use `r#macro`.

Reply:
Or you can use `macro_block`.

Reply:
> Or you can use `macro_block`.

Done.